src/__tests__/HealthCheckController.spec.ts

```diff
@@ -0,0 +1,110 @@
+import HealthCheckController from '../controllers/HealthCheckController';
+import mongoose from 'mongoose';
+
+jest.mock('mongoose');
+let connection: mongoose.Connection;
```

Comment:
Definitely should not be `let` because this variable never changed

Reply:
Rewrited: Now connection is set once on test run